adders/drizzle/config/tests.js

```diff
@@ -0,0 +1,100 @@
+import { defineAdderTests } from "@svelte-add/core";
+import { options } from "./options";
+
+const defaultOptionValues = {
+    sqlite: options.sqlite.default,
+    mysql: options.mysql.default,
+    postgresql: options.postgresql.default,
+    docker: options.docker.default,
+};
+
+export const tests = defineAdderTests({
+    options,
+    optionValues: [
+        { ...defaultOptionValues, database: "sqlite", sqlite: "better-sqlite3" },
+        { ...defaultOptionValues, database: "sqlite", sqlite: "libsql" },
+        { ...defaultOptionValues, database: "mysql", mysql: "mysql2", docker: true },
+        { ...defaultOptionValues, database: "postgresql", postgresql: "postgres.js", docker: true },
```

Comment:
Why do we have `defaultOptionValues`? Looks like we are re-defining all those options nevertheless. Or is it because we need to specify a valid option for `mysql` even if we are using `sqlite`?

Reply:
Default values are needed to satisfy the type of `optionValues`. Since the type for `options` doesn't use a discriminated union, nor are they considered 'optional', each db type must be explicitly defined/initialized. In this case, we're just defining them with their default state, which is an explicit `undefined`.